src/data.jl

```diff
@@ -522,12 +522,16 @@ end
 Base.getindex(d::DensityOfStates, ind) = (d.energy[ind], d.dos[ind], d.int[ind])
 
 # Setting up for gaussian smearing of DOS curve
-# Fourier transformed gaussian for convolution in smearing
 function gaussian(
```

Comment:
It might make more sense to define `gaussian(length::Integer, sigma::Real)`, and in the `smear()` function, use the length of the `DensityOfStates`. However, I don't think `Base.length()` is defined for `DensityOfStates` so this is probably not a big deal, especially since this isn't a public API.

Reply:
I'll probably refactor this code at some point but it shouldn't affect the current work.